okhttp/src/jvmMain/kotlin/okhttp3/Cookie.kt

```diff
@@ -492,8 +531,18 @@ class Cookie private constructor(
         path = if (lastSlash != 0) encodedPath.substring(0, lastSlash) else "/"
       }
 
+      // If SameSite is absent or not a valid option, use the default value.
+      if (sameSite == null) {
+        sameSite = "lax"
+      }
+
+      // If SameSite is set to `None`, Secure must be true
```

Comment:
I don’t think we should do this. The parsed `Cookie`’s job is to represent what was encoded only. If the user wants to break the rules, that’s not our business at this layer.